lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -62,10 +62,10 @@ class RadarChartPainter extends BaseChartPainter<RadarChartData> {
 
     dataSetsPosition = calculateDataSetsPosition(canvasWrapper.size, holder);
 
-    drawGrids(canvasWrapper, holder);
     drawTicks(context, canvasWrapper, holder);
     drawTitles(context, canvasWrapper, holder);
     drawDataSets(canvasWrapper, holder);
+    drawGrids(canvasWrapper, holder);
```

Comment:
Why did you change the order of this draw call?
Is there any specific reason we should know?